### ISSUE TITLE
Naming corrections needed in Experience Event

### ISSUE THREAD

**Issue by cdegroot-adobe** [adobe/xdm#274]
## What are the schemas that are affected by the issue
endUserIds
webInfo

## What are examples of products that are impacted by the issue
Target
Analytics.

Issue 1.
The naming inconsistency of the field for web page is bothersome:
https://github.com/adobe/xdm/blob/v0.9/schemas/context/webinfo.schema.json#L16
It's including the metric/"event" in the field name and we don't do that in the rest since those dimensions may be populated when it wasn't a page-view like wanting the page URL hosting the link that was clicked on.
The request is to change xdm:webPageView to xdm:webPage.
This needs to be fixed before we can have a 1.0.

Issue 2.
We have inconsistent naming of EndUserIDs fields for solutions:
https://github.com/adobe/xdm/blob/v0.9/schemas/context/enduserids.schema.json#L16-L36
We just need to pick one pattern and be consistent before we cut 1.0.

**Comment by trieloff** [comment]
I think these are two separate issues and should be filed and tracked separately. 

Changing `webPageView` to `webPage` and using it for non-pageload interactions is changing the semantics and would be a breaking change.

**Comment by lrosenthol** [comment]
We agreed for .9 that it would be webPageView -we're not changing it back (at least not without a very good justification)

**Comment by trieloff** [comment]
Closing for issue 1 (see https://github.com/adobe/xdm/issues/274#issuecomment-386065612). Please open a separate issue for issue 2.
